AjavaBasics/homeWork/homeWork7Arrays2/src/Main.java

```diff
@@ -1,5 +1,53 @@
+import java.util.Arrays;
+import java.util.Random;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        //Task 1
+        System.out.println("Task 1");
+        int[] expenses = {1_200_000, 1_500_000, 1_100_000, 1_300_000, 1_400_000};
+        int summ = 0;
+        for (int i : expenses) summ += i;
```

Comment:
Рекомендую использовать фигурные скобки всегда, даже если в теле цикла/условия всего одна инструкция
```suggestion
        for (int i : expenses) {
            summ += i;
         }
```